packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md

```diff
@@ -219,7 +219,11 @@ Run the static-spa sample application to see a working example of the authentica
 * Download and install the sample application.
 * Run the sample application.
 
-<img src="/img/okta-sign-in-javascript-custom.png" alt="Screenshot of a custom sign-in form used with the Auth JS SDK" width="400">
+<div class="half">
```

Comment:
This is a really good question to bring up @vanngo-okta . You are right that class names should be descriptive, and you have followed a good path towards best-practice, which is class names that describe the cotent, semantics, and/or effect of the class, rather than just the presentational result. In addition, the class names we've chosen might not be ambiguous now, but what about in the future if we decide to introduce some other set of classes for setting the height of something, or the opacity of something?

I'd be in favor of using some kind of convention along the lines of *contentstyled-propertyaffected-value*, so we can keep any future class name decisions consistent and at least semi-intuitive.

I think my one issue with your suggestion is that it is too specific — potentially these classes could be used to set the width of other blocks of content inside articles (videos, audio players, even specific text blocks?). I'd therefore like to suggest something more generic, like:

```
block-width-quarter
block-width-half
block-width-three-quarter
block-width-full
```

I'd also like to suggest that we don't block this work on choosing final classnames. It will be pretty quick to do a search and replace if required, once the discussion is had, and in the meantime I'd like to push the work forward.

Thoughts?

Reply:
Agreed. This is not blocking, just a suggestion.